components/host-dashboard/reports-section/helpers.js

```diff
@@ -5,6 +5,11 @@ export const getActiveYearsOptions = host => {
   return activeYears.map(year => ({ value: year, label: year })).reverse();
 };
 
-export const formatAmountForLegend = value => {
-  return value < 1000 ? value : `${Math.round(value / 1000).toLocaleString()}k`;
+export const formatAmountForLegend = (locale, currency, value) => {
```

Comment:
value should stay first argument as it was before, this is what you format, the rest are options.

```suggestion
export const formatAmountForLegend = (value, currency, locale) => {
```

Reply:
It's not acceptable to integrate this change and mark it as "resolved" without updating the places where it's used. I expect that you think about this yourself.

Reply:
Sorry about this, I thought in my mind I updated the usages; should have double checked it better. 🤯